src/utils/primitive.js

```diff
@@ -1,24 +1,28 @@
-const childProcess = require('child_process')
-const fs = require('fs')
-const path = require('path')
-const os = require('os')
+import path from 'path'
+import os from 'os'
+
+import execa from 'execa'
+import fs from 'fs-extra'
 
 const VENDOR_DIR = path.resolve(__dirname, '..', '..', 'vendor')
-const PRIMITIVE_TEMP_FILE = os.tmpdir() + '/primitive_tempfile.svg'
+const PRIMITIVE_TEMP_FILE = path.resolve(
+  os.tmpdir(),
+  `primitive-tempfile-${new Date().getTime()}.svg`
```

Comment:
I recommend using https://www.npmjs.com/package/tempfile here to avoid possible tempfile clashes when many files are worked on in parallel

Reply:
Cool idea!! Maybe even https://github.com/sindresorhus/tempy since it is recommended in the tempfile README.md :)

Reply:
Done, see #45